crates/fj-core/src/services/service.rs

```diff
@@ -62,6 +62,12 @@ impl<S: State> Deref for Service<S> {
     }
 }
 
+impl<S: State> DerefMut for Service<S> {
```

Comment:
It feels wrong, but I don't have a better idea right now. Since it's only used in tests, we should slap a `#[cfg(test)]` on there, to make sure it's not misused by other code.